sdk/core/azure_core_amqp/Test-Setup.ps1

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# cspell: ignore JOBID
+
+param (
+  [string]$PackageName
+)
+
+# Load common ES scripts
+. "$PSScriptRoot\..\..\..\eng\common\scripts\common.ps1"
+
+$WorkingDirectory = ([System.IO.Path]::Combine($RepoRoot, "../TestArtifacts"))
+
+if (-not $PackageName) {
+  Write-Host "PackageName parameter not provided."
+  exit 1
+}
+
+# Create the working directory if it does not exist.
+Write-Host Using Working Directory $WorkingDirectory
```

Comment:
It is odd to see unquoted strings going to a powershell command

```suggestion
Write-Host "Using Working Directory $WorkingDirectory"
```

Reply:
Crud - missed that one.